mod_test.ts

```diff
@@ -1126,3 +1192,55 @@ Deno.test("jsr specifiers - no lockfile", async (t) => {
     assertEquals(ns.join("a", "b"), join("a", "b"));
   });
 });
+
+Deno.test("ignores local css, uses default loader", async (t) => {
+  await testLoader(t, LOADERS, async (esbuild, loader) => {
+    if (esbuild === PLATFORMS.wasm) return;
+    const res = await esbuild.build({
+      ...DEFAULT_OPTS,
+      format: undefined,
+      plugins: [...denoPlugins({ loader })],
+      entryPoints: ["./testdata/basic.css"],
+      bundle: true,
```

Comment:
Enabling bundle makes it try resolving paths in the css file, and shows that the denoPlugins won't try to resolve or load the svg data url that is being used as a background image.